arch/risc-v/src/esp32c3/Kconfig

```
@@ -402,6 +402,12 @@ config ESP32C3_ONESHOT
 	---help---
 		Enable a wrapper around the low level timer/counter functions to
 		support one-shot timer.
+config ESP32C3_FREERUN
```

```suggestion
		support one-shot timer.

config ESP32C3_FREERUN
```
Add a blank line between options

---

@gustavonihei done